src/products/firefox_android/how_to_localize.md

```diff
@@ -1,35 +1,17 @@
 # Firefox for Android - How to localize
 
-The release cycle for Firefox for Android is the same as Firefox for Desktop, which is generally 6 weeks - but can sometimes be 8 weeks.
+Firefox for Android is on a four week rapid release cycle. This means that usually there is a new release going out every four weeks.
 
-You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
+Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that you can complete your work. The deadline in Pontoon is an indicator of when the next localization cut-off date is for the upcoming release.
```

Comment:
> localizers will… you can complete your work

The latter should keep using "they".

I think you should also explain what "cut-off date" means in this context. Is the last day to get localization updates into that release?

Reply:
Done!